main.tf

```
@@ -103,11 +103,15 @@ resource "aws_cloudwatch_event_target" "this" {
     ] : []
 
     content {
-      group               = lookup(ecs_target.value, "group", null)
-      launch_type         = lookup(ecs_target.value, "launch_type", null)
-      platform_version    = lookup(ecs_target.value, "platform_version", null)
-      task_count          = lookup(ecs_target.value, "task_count", null)
-      task_definition_arn = lookup(ecs_target.value, "task_definition_arn", null)
+      group                   = lookup(ecs_target.value, "group", null)
+      launch_type             = lookup(ecs_target.value, "launch_type", null)
+      platform_version        = lookup(ecs_target.value, "platform_version", null)
+      task_count              = lookup(ecs_target.value, "task_count", null)
+      task_definition_arn     = lookup(ecs_target.value, "task_definition_arn", null)
```

This is required argument, so it should be like this:

```suggestion
      task_definition_arn     = ecs_target.value.task_definition_arn
```